### DISCUSSION TITLE
Use OneDrive to host

### DISCUSSION THREAD

**Discussion by tedomi2705** [spencerwooo/onedrive-vercel-index#1044] [Q&A]
Can i use this to host static website or use my onedrive as CDN for my website?

**Comment by spencerwooo** [comment]
I believe so, tho your speed won't be guaranteed.

**Reply by tedomi2705** [reply]
Can i show only publiced file?

**Reply by tedomi2705** [reply]
and i dont want html file to get downloaded too
